federationapi/routing/join.go

```diff
@@ -137,9 +162,10 @@ func SendJoin(
 	}
 
 	// Check that the event is signed by the server sending the request.
+	redacted := event.Redact()
```

Comment:
This one needs to be separate to avoid `JSON is not in method set of gomatrixserverlib.Event`. The function takes a pointer receiver but returns a copy.